### ISSUE TITLE
Frequency selection reverts when creating new deployments

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#612]
I made deployments today for 3 SGs. I had selected Mixed frequency then specified the frequency for each port. However upon saving, the "Mixed" selection reverted to "166", along with the frequency setting for the port that was listening on. When edited the second time,  however, the frequency settings stuck. This occurred with each of the 3. 

Just now I created a 4th test deployment and observed the same behaviour.

<br/>
<br/>
<br/>

Settings prior to hitting "save":

![2019-11-11_170552](https://user-images.githubusercontent.com/55001144/68624883-da7d9680-04a5-11ea-81c4-5dfc505a6e42.png)

<br/>
<br/>
<br/>

Settings after hitting "save":

![2019-11-11_170610](https://user-images.githubusercontent.com/55001144/68624921-eb2e0c80-04a5-11ea-9a84-1e92835ba92e.png)
<br/>
<br/>
<br/>

After changing them back and saving a second time, the settings stuck:

![2019-11-11_170633](https://user-images.githubusercontent.com/55001144/68624960-039e2700-04a6-11ea-89f9-0f74d69e4079.png)

**Comment by pmorrill** [comment]
I've put a fix for this into beta9, so you can test it tomorrow on beta server.

**Comment by jsayers-bsc** [comment]
I just tested this with several antenna modifications and it seems to be resolved. 

However, I just noticed a similar issue that may or may not be related.

Modifying the Receiver Frequency for the deployment will overwrite the antenna-specific values with the new selection regardless of the previous setting or dongle type. I should note, this only in Live so the fix may just waiting to come over from Beta.
